cpp/net/connection_pool.cc

```diff
@@ -30,20 +132,57 @@ const HostPortPair& ConnectionPool::Connection::other_end() const {
 
 
 ConnectionPool::ConnectionPool(libevent::Base* base)
-    : base_(CHECK_NOTNULL(base)), cleanup_scheduled_(false) {
+    : base_(CHECK_NOTNULL(base)),
+      cleanup_scheduled_(false),
+      ssl_ctx_(CHECK_NOTNULL(SSL_CTX_new(TLSv1_client_method())),
+               SSL_CTX_free) {
+  // Try to load trusted root certificates.
+  // TODO(alcutter): This is probably Linux specific, we'll need other sections
```

Comment:
Not really; I could see the flag being set to `System` (or whatever) on OS X/Pyramid OS/...  (or another `FLAG_use_system_ca_roots`  - bearing in mind that you may wish to NOT use the system roots and instead load them from a file.)
But the code for what `System` would do is not going to be this code (unless you're running Debian) and *that's* what this comment is about and why it's here and not on the flag.